test/integration/policy_generator_helm_test.go

```diff
@@ -0,0 +1,133 @@
+// Copyright Contributors to the Open Cluster Management project
+
+package integration
+
+import (
+	"context"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+
+	"github.com/stolostron/governance-policy-framework/test/common"
+)
+
+var _ = Describe("GRC: [P1][Sev1][policy-grc] Test the Policy Generator "+
+	"with a Helm chart", Ordered, Label("SVT"), func() {
+	const policyName = "e2e-grc-helm-policy-app"
+	const namespace = "grc-e2e-helm-policy-generator"
+
+	It("Sets up the application subscription", func() {
+		By("Creating the application subscription")
+		_, err := common.OcUser(
+			gitopsUser,
+			"apply",
+			"-f",
+			"../resources/policy_generator/subscription-helm.yaml",
+			"-n",
+			namespace,
+		)
+		Expect(err).ShouldNot(HaveOccurred())
+
+		Eventually(func(g Gomega) {
+			appSubRsrc, err := clientHubDynamic.Resource(common.GvrSubscription).Namespace(namespace).Get(
+				context.TODO(), "grc-e2e-helm-policy-generator-subscription", metav1.GetOptions{},
+			)
+			g.Expect(err).ShouldNot(HaveOccurred(), "The subscription should exist.")
+
+			appSubPhase, found, err := unstructured.NestedString(appSubRsrc.Object, "status", "phase")
+			g.Expect(err).ShouldNot(HaveOccurred(), "The subscription status should be parseable.")
+			g.Expect(found).Should(BeTrue(), "The subscription status should have a phase.")
+			g.Expect(appSubPhase).Should(Equal("Propagated"), "The subscription should propagate successfully.")
+		},
+			defaultTimeoutSeconds,
+			1,
+		).Should(Succeed())
+	})
+
+	It("Validates the propagated policies", func() {
+		// Perform some basic validation on the generated policy.
+		By("Checking that the root policy was created")
+		policyRsrc := clientHubDynamic.Resource(common.GvrPolicy)
+		var policy *unstructured.Unstructured
+		Eventually(
+			func() error {
+				var err error
+				policy, err = policyRsrc.Namespace(namespace).Get(
+					context.TODO(), policyName, metav1.GetOptions{},
```

Comment:
Optional: ctx SpecContext
```suggestion
					ctx, policyName, metav1.GetOptions{},
```